google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java

```diff
@@ -303,6 +304,63 @@ public void crossInstanceRestoreTest()
     }
   }
 
+  @Test
+  public void crossInstanceCopyBackupTest()
```

Comment:
I would add an additional copy backup test that is in the same instance.

Reply:
Done.